docs/community/old-setup/old-hostsystem.md

```diff
@@ -1,169 +0,0 @@
----
-title: Host system
-layout: default
-parent: Community setup
-grand_parent: Community
----
-
-# Host System
-
-## Operating System Configuration
-
-This page discusses software configuration issues commonly encountered when deploying FeatureBase. For hardware requirements or scaling, look at [Sizing FeatureBase](/docs/community/com-config/old-size-featurebase-database).
-
-FeatureBase's data storage splits the database into a potentially large number of files, which are open simultaneously. Most Unix-derived systems limit the number of simultaneous open files for a given process or user, and these limits are frequently small enough to be a problem. Similarly, there may be limits on the number of memory-mapped regions allowed in a process. These may need to be changed,
-either temporarily during initial development and testing, or as a machine-wide configuration change that persists across restarts.
-
-Throughout these instructions, commands given with a root prompt (`#`) are expected to require root privileges; you can run them in a root shell, or using `sudo`.
-
-### Linux Systems
-
-You can make temporary changes just to run the software once, or persistent changes which will survive reboots. We document both because the one-off changes are much easier to make, but in a production environment you will be happier making persistent changes.
-
-To temporarily increase the open file limit, you should be able to use `ulimit -n`. If you can't set the limit high enough, you will need root privileges. The `ulimit` program only affects the limits of the shell it's run in and child processes of that shell, so you can't use `sudo ulimit` to change the limit of your current shell; if you need to set a higher limit, launch a new shell using `sudo -s` or the equivalent, then change the ulimit, then use `su` to start a new shell running as your regular user but inheriting the higher limits. We do not recommend running FeatureBase with root privileges.
-
-To temporarily increase the allowable memory-mapped regions on a Linux system, you can use the `sysctl` command, or just the `proc` filesystem:
-
-```shell
-# sysctl -w vm.max_map_count=N
-# echo N > /proc/sys/vm/max_map_count
-```
-
-#### Persistent Changes
-
-There are two likely paths to setting the system open files limit on Linux systems. One is to look at `/etc/security/limits.conf`, or `/etc/security/limits.d/*.conf`, which allow you to specify the `hard` limit for the item `nofile` for any user or group. The other is to use `sysctl`, and change the `fs.file-max` variable. This can be changed through `/etc/sysctl.conf` or `/etc/sysctl.d/*.conf`. You can also use the `sysctl.conf` approach to change `vm.max_map_count` persistently.
-
-Either way, you will need to restart your login session for the change to take effect, but it should be persistent after that. We recommend an open file limit of around 256K by default.
-
-#### Enable noatime
```

Comment:
why was this content no included in the new stuff?

Reply:
As above

Reply:
done